dask/dataframe/io/tests/test_parquet.py

```diff
@@ -3853,3 +3853,30 @@ def _prep(x):
     df_read_dask = dd.read_parquet(pa_path, engine=engine)
     df_read_pa = pq.read_table(pa_path).to_pandas()
     assert_eq(_prep(df_read_dask), _prep(df_read_pa), check_index=False)
+
+
+@pytest.mark.parametrize("filter_value", ({1}, [1], (1,)), ids=("set", "list", "tuple"))
+def test_in_predicate_can_use_iterables(tmpdir, engine, filter_value):
+    """Fixes GH8720"""
```

Comment:
Small (completely optional) comment, but I've found it quite useful to have the full issue URL available in these contexts. It makes it easier to get directly to the related issue 

```suggestion
    """Regression test for https://github.com/dask/dask/issues/8720"""
```